### PR TITLE
Task/add top level make

### PR DESCRIPTION
**Pull request by badisa**

### CODE REVIEW

**Comment by badisa** [comment]
The intention is for the make file to be called out to by CI, that way we have more control over what CI is doing and a way to investigate without impacting everyone.

The Jenkins builds this is being built against is also slightly modified in an attempt to properly clean up the jenkin's dirs.

**Review by maxentile — APPROVED** [review]
Nice! A clear improvement, and works like a charm

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+INSTALL_PREFIX := $(MKFILE_DIR)timemachine/
+
+NPROCS = `nproc`
+
+CUDA_ARCH := "sm_70"
+
+.PHONY: build
+build:
+	mkdir -p $(CPP_DIR)build/ && cd $(CPP_DIR)build/ &&  \
+	cmake -DCUDA_ARCH=$(CUDA_ARCH) -DCMAKE_INSTALL_PREFIX=$(INSTALL_PREFIX) ../ && \
```
Comment (proteneer) [inline Makefile:15]: Would it be possible to enable setting CUDA_ARCH from the commandline or as a env variable somehow?

File: README.md [hunk 2]
```diff
@@ -47,11 +47,7 @@ conda install -c conda-forge -c openeye openmm openeye-toolkits rdkit=2021.03.1
 
 ```
 pip install -r requirements.txt
-cd timemachine/cpp
-mkdir build
-cd build
-cmake -DCUDA_ARCH=sm_70 ../
-make -j4 install
+make CUDA_ARCH=sm_70 build
```
Comment (proteneer) [inline README.md:50]: Does this override the `:=` in the Makefile?

Comment (badisa) [reply README.md:50]: It does

**Comment by proteneer** [comment]
lgtm

**Review by proteneer — APPROVED** [review]

**Pull request merged**